ts/src/pro/binanceusdm.ts

```diff
@@ -19,6 +19,7 @@ export default class binanceusdm extends binance {
                 'defaultSubType': 'linear',
             },
             // https://binance-docs.github.io/apidocs/futures/en/#error-codes
+            // https://developers.binance.com/docs/derivatives/usds-margined-futures/error-code
```

Comment:
@Dan-krm can we add the docs to the pro/binance.ts?

Reply:
Yeah I added a doc link to the pro implementations